modules/attribution-document/attribution-document-generator/src/main/java/org/eclipse/sw360/antenna/attribution/document/workflow/generators/ArtifactAdapter.java

```diff
@@ -63,7 +63,12 @@ public static String getLicenseText(License license) {
       String text = license.getText();
       text = StringUtils.isNotBlank(text) ? text : "No license text available";
       // remove the magic unicode as it will likely not be in a unicode font set.
-      return StringUtils.replace(text, "\uFEFF", "");
+      text = text
+              .replace("\u0009", "    ")
+              .replace("\u00a0", " ")
+              .replace("\uFEFF", "")
```

Comment:
Nit: We should align on either lower-case or upper-case hex-digits. Personally, I prefer lower-case.

Reply:
Another nit: This should come last for proper sorting of hex-codes.

Reply:
Done.